getting_started/first_3d_game/02.player_input.rst

```diff
@@ -129,7 +129,7 @@ Bind also the :kbd:`A` key, onto the action ``move_left``.
 |image12|
 
 Let's now add support for a gamepad's left joystick. Click the "+" button again
-but this time, select *Manual Selection -> Joypad Axes*.
```

Comment:
`Manual  Selection` is no longer a tab, i uploaded a new `getting_started/first_3d_game/img/02.player_input/joystick_axis_input.webp` image to match it.